core/src/main/java/org/eclipse/microprofile/reactive/streams/core/InternalStages.java

```diff
@@ -63,4 +70,33 @@ ReactiveStreamsGraphBuilder getBuilder() {
             return graphBuilder;
         }
     }
+
+    /**
+     * A nested stage, holding a graph. This is used when nesting a builder that has come from another implementation.
+     */
+    static final class NestedGraph implements InternalStage {
```

Comment:
It is interesting that we spent some time on last weeks call discussing why a
Graph does not have an 'isa' relationship to a Stage.